### ISSUE TITLE
Autolayout errors when in-call statusbar is visible

### ISSUE THREAD

**Issue by otymartin** [uias/Pageboy#61]
@msaps 

When in-call StatusBar/hotSpot StatusBar is present (On device, havent tried simulator), the following auto-layout errors are given by the compiler.

This can be recreated on the pageboy example app by just turning on personal hotspot and launching the app on device.

```javascript
2017-05-30 11:51:07.328140+0300 Pageboy-Example[2099:469758] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x170098ba0 h=-&- v=-&- _UIPageViewControllerContentView:0x100d08a20.midY == UIView:0x100d12150.midY + 20   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x170098bf0 h=-&- v=-&- _UIPageViewControllerContentView:0x100d08a20.height == UIView:0x100d12150.height   (active)>",
    "<NSLayoutConstraint:0x170096a80 V:|-(0)-[_UIPageViewControllerContentView:0x100d08a20]   (active, names: '|':UIView:0x100d12150 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x170096a80 V:|-(0)-[_UIPageViewControllerContentView:0x100d08a20]   (active, names: '|':UIView:0x100d12150 )>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
2017-05-30 11:51:07.334525+0300 Pageboy-Example[2099:469758] [LayoutConstraints] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
	(Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSAutoresizingMaskLayoutConstraint:0x170098ba0 h=-&- v=-&- _UIPageViewControllerContentView:0x100d08a20.midY == UIView:0x100d12150.midY + 20   (active)>",
    "<NSAutoresizingMaskLayoutConstraint:0x170098bf0 h=-&- v=-&- _UIPageViewControllerContentView:0x100d08a20.height == UIView:0x100d12150.height   (active)>",
    "<NSLayoutConstraint:0x170096e40 V:[_UIPageViewControllerContentView:0x100d08a20]-(0)-|   (active, names: '|':UIView:0x100d12150 )>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x170096e40 V:[_UIPageViewControllerContentView:0x100d08a20]-(0)-|   (active, names: '|':UIView:0x100d12150 )>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
```

**Comment by msaps** [comment]
Looks like something interesting inside `UIPageViewController` having some autoresizing masks when it potentially shouldn't. I'll take a look! 👍

**Comment by msaps** [comment]
@otymartin this should now be resolved as of [1.1.0](https://github.com/uias/Pageboy/releases/tag/1.1.0) 😄

**Comment by otymartin** [comment]
@msaps Nice thanks man :)
